### ISSUE TITLE
100% width

### ISSUE THREAD

**Issue by meshesha** [nolanlawson/emoji-mart-outside-react#11]
Is there a way I can make the width 100%.
i added `.emoji-mart { width: 100% !important }` in file [css/emoji-mart.css](https://github.com/missive/emoji-mart/blob/master/css/emoji-mart.css).
but there issue with categories link;

**Comment by rugk** [comment]
I basically have a fork of that in https://github.com/rugk/emoji-mart-embed

And I use it in a Firefox add-on, where you can select your width in the settings. This adjusts an option of emoji-mart so you have the width you want then.

Apart from that this CSS may help: https://github.com/rugk/awesome-emoji-picker/blob/master/src/popup/index.css
Note you can also add styles via [the style setting](https://github.com/rugk/awesome-emoji-picker/blob/fe26e5f4d2e067d7e1f84b4718d9f947484b4e83/src/popup/module/EmojiPicker.js#L25)'

Because the browser context has some quirks, [here is some CSs applied via JS](https://github.com/rugk/awesome-emoji-picker/blob/fe26e5f4d2e067d7e1f84b4718d9f947484b4e83/src/popup/index.js#L87-L105) that also changes the width.
So that should all be possible.

Basically, now having gone through my own code again [this here should do the trick](https://github.com/rugk/awesome-emoji-picker/blob/fe26e5f4d2e067d7e1f84b4718d9f947484b4e83/src/popup/index.css#L12-L18).

**Comment by meshesha** [comment]
Thank you very much.
